warrior/Actions/CommonActions/common_actions.py

```diff
@@ -198,6 +200,16 @@ def verify_key_already_exists_and_update(orig_dict, new_dict):
                 filepath = getAbsPath(filepath, os.path.dirname(testcasefile_path))
                 with open(filepath, "r") as json_handle:
                     json_doc = json.load(json_handle)
+                    #if slurp is set to True, all content of given json file will be
+                    #store in data repository
+                    if slurp:
```

Comment:
What happens if I have the following in the testcase?
```<argument name="slurp" value="False"/>```

Reply:
this won't store the content from given json, will add negative tests to clarify this